components/sysutils/slurm/files/getgrouplist.h

```diff
@@ -0,0 +1,96 @@
+/*-
```

Comment:
Why do we need it? Just to generate patches/11-getgrouplist.patch ? Then there's issue... getgrouplist.c is BSD-licensed, slurm - GPLv2-licensed. We can't mix such code in one file and I'm not sure if we can link such files in one object file.